test/new-e2e/agent-subcommands/flare/flare_test.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package flare contains helpers and e2e tests of the flare command
+package flare
+
+import (
+	_ "embed"
+	"strings"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/fakeintake/client/flare"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
+	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
+	"github.com/stretchr/testify/assert"
+)
+
+type commandFlareSuite struct {
+	e2e.Suite[e2e.FakeIntakeEnv]
+}
+
+func TestFlareSuite(t *testing.T) {
+	e2e.Run(t, &commandFlareSuite{}, e2e.FakeIntakeStackDef(nil), params.WithDevMode())
```

Comment:
<!-- Warning highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes and they are blockers. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question. -->
**🔨 warning**
You should pass `params.WithDevMode` in local runs. You could have a slice of options and append `WithDevMode` only when `CI` env var is false

Reply:
I plan to remove it when I got all needed approvals

Reply:
> You could have a slice of options and append WithDevMode only when CI env var is false

Could it be something added directly into the framework ?

Reply:
Yes, absolutely !